### DISCUSSION TITLE
能否从传感器中选择某项作为温度显示？

### DISCUSSION THREAD

**Discussion by Canary233** [lollipopkit/flutter_server_box#312] [Ideas]
我的两套AMD平台都无法正常显示温度，实际温度在传感器那栏里

**Comment by Qs315490** [comment]
我的设备是使用sensors命令获取温度的，手机终端界面默认显示第一项（amd显卡电压），解决方案是修改 `.config/server_box/mobile_v44.sh` 文件中 `sensors -j` 这一行在后面加 `空格` 和 对应传感器名称，只显示这个传感器。但是不久后文件就会刷新，导致修改还原
